src/Rootfs.jl

```diff
@@ -631,9 +639,11 @@ function choose_shards(p::AbstractPlatform;
         )
 
         # We _always_ need Rootfs and PlatformSupport for our target, at least
+        # We don't have old platform support for riscv64. Remove this once all platfor support is aligned in time.
```

Comment:
Is this still necessary after https://github.com/JuliaPackaging/BinaryBuilderBase.jl/commit/dafa99380c04a0de80e1b478a3517839bc494dc1, since that's the only relevant version?

Reply:
Yes. BinaryBuilderBase chooses `PlatformSupport` by date, and it wants specifically `v"2024.08.10"`. That's different from `Rootfs` where it chooses the newest available one.